kedro-airflow/.pre-commit-config.yaml

```diff
@@ -32,21 +36,21 @@ repos:
         name: "Quick PyLint on kedro_airflow/*"
         language: system
         types: [file, python]
-        files: ^kedro_airflow/
+        files: ^kedro-airflow/kedro_airflow/
```

Comment:
Why is this one needed? Isn't the directory already changed to `kedro-airflow` when we call the `pre-commit` hook?